addons/spreadsheet/static/src/chart/plugins/chart_odoo_menu_plugin.js

```diff
@@ -16,10 +15,17 @@ export default class ChartOdooMenuPlugin extends CorePlugin {
     handle(cmd) {
         switch (cmd.type) {
             case "LINK_ODOO_MENU_TO_CHART":
-                this.history.update("odooMenuReference", cmd.chartId, cmd.odooMenuId);
-                break;
-            case "DELETE_FIGURE":
-                this.history.update("odooMenuReference", cmd.id, undefined);
+                {
```

Comment:
really nitpick: the bracket is functionally useless, and this prettying of a line with a single bracket is ugly IMO.